components/mediators/datamapper/org.wso2.carbon.mediator.datamapper/src/main/java/org/wso2/carbon/mediator/datamapper/DataMapperMediator.java

```diff
@@ -476,22 +476,22 @@ private Map getPropertiesMap(List<String> propertiesNamesList, MessageContext sy
             switch (contextAndName[0].toUpperCase()) {
```

Comment:
It will be better to define these context and name indexes as constants with names as INDEX_OF_CONTEXT and INDEX_OF_NAME.